### ISSUE TITLE
Widget selection

### ISSUE THREAD

**Issue by balandinodidonato** [BirminghamConservatoire/IntegraLive#832]
Maybe the module creator needs the possibility to select more widget with the mouse or clicking by the mouse and holding the shift key.

**Comment by elmiquito** [comment]
Yes, definitely. The Widget Layout view in Module Creator needs a select, copy and paste functionality to improve the workflow.
